queue_services/account-mailer/src/account_mailer/enums.py

```diff
@@ -48,6 +49,7 @@ class SubjectType(Enum):
     PAD_SETUP_FAILED = '[BC Registries and Online Services] Your Account is Temporarily Suspended'
     PAYMENT_PENDING = '[BC Registries and Online Services] Payment is now due for pending transaction on your account'
     EJV_FAILED = 'GL disbursement failure for EJV'
+    RESET_PASSCODE = 'Passcode reset'
```

Comment:
do you wanna add the [BC Registries and Online Services] in the header?

Reply:
I have asked subject and some header details from Roy in the ticket. Once he confirms, will change it. What I put there is only dummy.